fundingmanager.go

```diff
@@ -1833,6 +1841,10 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 		PendingChanId: pendingChanID[:],
 	}
 
+	// Inform the ChannelNotifier that the channel has entered
+	// pending open state.
+	f.cfg.NotifyPendingOpenChannelEvent(*fundingPoint)
```

Comment:
Should we move this into the select below? The `resCtx.updates` channel is where we report the upstream client, so it would make sense to notify the pending channel on send into that channel?

Reply:
Something like this:
```
	select {
	case resCtx.updates <- upd:
		// Inform the ChannelNotifier that the channel has entered
		// pending open state.
		f.cfg.NotifyPendingOpenChannelEvent(*fundingPoint)
	case <-f.quit:
		return
	}
```

Reply:
@carlaKC Thank you.
Fixed!